feat(cli): Format Option for list-apps

gavindsouza opened the pull request:
Now `bench list-apps` supports text and JSON outputs :)

![Screenshot 2021-04-22 at 5 59 39 PM](https://user-images.githubusercontent.com/36654812/115714243-882bef00-a394-11eb-9036-b66de5ceb48b.png)

- [x] Docs
- [x] Test

<!-- no-docs -->

gavindsouza committed:
feat(cli): Format Option for list-apps

surajshetty3416 commented on frappe/commands/site.py:
```diff
@@ -234,18 +237,25 @@ def fix_whitespaces(text):
+	if format == "json":
+		import json
+
+		print(json.dumps(summary_dict))
```
use `click.echo`?

`print` here looks like a debug statement 
```suggestion
		click.echo(json.dumps(summary_dict))
```

gavindsouza committed:
test: Added tests for bench list-apps -f json

gavindsouza committed:
chore: Re-trigger CI builds

gavindsouza committed:
chore: Use click.echo in favour of print

Accepting suggested changes because `click.echo` has improved support for handling Unicode and binary data that "does not fail no matter how badly configured the system is" _(ref: docstring)_.

Co-authored-by: Suraj Shetty <13928957+surajshetty3416@users.noreply.github.com>

surajshetty3416 committed:
Merge branch 'develop' into list-apps-update

gavindsouza commented:
@Mergifyio backport version-13-hotfix
